packages/config/config/devices/0x015f/mh-3901z.json

```diff
@@ -0,0 +1,129 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH3901-Z",
+	"description": "Boiler Thermostat",
+	"devices": [
+		{
+			"productType": "0x3910",
+			"productId": "0x5102"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Temperature reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "Difference reporting only",
+					"value": 1
+				},
+				{
+					"label": "Interval reporting only",
+					"value": 2
+				},
+				{
+					"label": "Difference and interval reporting",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Temperature difference setting",
+			"description": "Based on 0.1 °C unit: n * 0.1 °C = 0.5 °C",
```

Comment:
Unnecessary decription, this information is encoded in the `unit`.

Reply:
removed description